src/components/pages/room.js

```diff
@@ -1,14 +1,59 @@
-import React from 'react';
 import PropTypes from 'prop-types';
-import { Row, Space, List } from 'antd';
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useDateFormat } from '../language-context.js';
-import { roomShape } from '../../ui/default-prop-types.js';
+import { Row, Space, List, Collapse, Button } from 'antd';
+import { invitationShape, roomShape } from '../../ui/default-prop-types.js';
+import RoomInvitationCreationModal from '../room-invitation-creation-modal.js';
 
 export default function Room({ PageTemplate, initialState }) {
   const { t } = useTranslation('room');
   const { formatDate } = useDateFormat();
-  const { roomDetails } = initialState;
+  const { roomDetails, invitations } = initialState;
+  const [isInvitationModalOpen, setIsRoomInvitationModalOpen] = useState(false);
```

Comment:
Usually with `useState` the naming pattern is `state`, `setState`.
In this case the pattern is broken.

Reply:
I would like to correct this pattern.

In previous projects I was using the state to set different state related fields (usually our states are quite small).
For example here if we would have a second thing that we would need on the state we would have:
const [secondThing, setSecondThing] = useState(...);
This allows us to setSecondThing without doing the
setState(prevState => ({...preveState, secondThing: {}));

This way of working is also in tune with what the documentation of useState says:

What do we pass to useState as an argument? The only argument to the useState() Hook is the initial state. Unlike with classes, the state doesn’t have to be an object. We can keep a number or a string if that’s all we need. In our example, we just want a number for how many times the user clicked, so pass 0 as initial state for our variable. **(If we wanted to store two different values in state, we would call useState() twice.)**

[Link to the documentation](https://reactjs.org/docs/hooks-state.html)

What do you guys thing?

Reply:
If you agree, I would also change the pattern on the room invitation modal and we can change the rest as we go.

Reply:
Ok, sorry, I see you took my remark quite literally, as I used the documentation terminology. I didn't literally mean `state`, just the actual pattern of `something` and `setSomething`.
And in your code this is not correct.

Reply:
Regarding splitting up a state in multiple states, I would not like to be black or white on it, as we should use our best judgement depending on case. In some cases, the components should stay together (even if they are not always set at the same time) because, due to the async manner of the state being set, it is a way to guarantee state consistency between events.